jeta/archive/ingest.py

```diff
@@ -203,32 +213,36 @@ def calculate_delta_times(msid, times, epoch=None):
     _times[msid] = np.diff(np.insert(jd_times, 0, epoch))
 
 
-def sort_msid_data_by_time(msid, times=None, values=None, append=True):
+def sort_msid_data_by_time(mid, times=None, values=None, append=True):
 
     global _times
     global _values
 
     if append:
-        _times[msid] = np.append(_times[msid], times)
-        _values[msid] = np.append(_values[msid], values)
-
-    msid_times = _times[msid]
-    msid_values = _values[msid]
+        pass
+        # FIXME: this may not be require with the new design
+        # _times[mid] = np.append(_times[mid], times)
+        # _values[mid] = np.append(_values[mid], values)
 
-    idxs = torch.argsort(torch.from_numpy(msid_times))
-
-    _times[msid] = np.array(msid_times[idxs])
-    _values[msid] = np.array(msid_values[idxs])
+    idxs = torch.argsort(torch.from_numpy(_times[mid]))
 
+    _times[mid] = _times[mid][idxs]
+    _values[mid] = _values[mid][idxs]
 
 def _sort_ingest_files_by_start_time(list_of_files=[]):
```

Comment:
the log message is printed before the list is sorted.  Suggest sorting the list, printing the log message, and then returning the already sorted list.